extension/src/popup/components/accountHistory/HistoryItem/index.tsx

```diff
@@ -138,6 +138,8 @@ export const HistoryItem = ({
 
   const renderBodyComponent = () => BodyComponent;
   const renderIcon = () => IconComponent;
+  /* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
@piyalbasu maybe I'm missing something but this rule doesn't seem right. It wants to include `t` as a dep of the `useCallback` hook here but really we want an empty dep array. 
We want a memoized version of `t` for the whole lifecycle of the component since it never changes and we need it as a dep of useEffect, but adding `t` as a dep of the callback  or effect causes an infinite loop in the tests.
https://react.dev/reference/react/useCallback#parameters

Not sure why we don't see an infinite loop when actually running the component and only in Jest though 🤔